type-helpers.ts

```diff
@@ -0,0 +1,10 @@
+export type PrettyPrint<T> = T extends infer U extends object
+  ? { [K in keyof U]: U[K] }
+  : T extends object
+  ? never
+  : T;
+
+export const isin = <T extends object>(
```

Comment:
Neat

Reply:
Ain't it? I learned that a few weeks ago. Super helpful for making types less ugly/archaic